repos/fdbt-netex-output/src/netex-convertor/period-tickets/periodTicketNetexHelpers.ts

```diff
@@ -938,7 +938,7 @@ export const getPeriodConditionsElement = (
     product: { productName: string; productValidity?: string },
 ): NetexObject => {
     let usagePeriodValidity = {};
-    const type = userPeriodTicket.type;
+    const type = 'single';
```

Comment:
Do we need this variable? I think it would be clearer if we just put the word 'single' where it is needed. We also dont need the startCase() function on it, we can just use 'Single'.